freqtrade/persistence/migrations.py

```diff
@@ -160,7 +161,7 @@ def check_migrate(engine, decl_base, previous_tables) -> None:
     table_back_name = get_backup_name(tabs, 'trades_bak')
 
     # Check for latest column
-    if not has_column(cols, 'open_trade_value'):
+    if not has_column(cols, 'open_trade_value') or not has_column(cols, 'buy_signal_name'):
```

Comment:
there can only be one last column. If that last column exists, no migration is necessary.
As such - this should look (roughly?) as follows:
```suggestion
    if not has_column(cols, 'buy_signal_name'):
```